Renegade X Launcher/SelfUpdater.cs

```diff
@@ -134,12 +134,15 @@ public static void ExecuteInstall()
                         "cd /D \"" + Path.GetTempPath() + "\"",
 
                         // Wait for the launcher to close.
-                        ":wait_for_close",
-                        "tasklist /FI \"PID eq " + pidString + "\" /FO csv /NH | find \"\"\"" + pidString + "\"\"\" > nul",
-                        "if not errorlevel 1 (",
-                        "    timeout /t 1 > nul",
-                        "    goto :wait_for_close",
-                        ")",
+                        //":wait_for_close",
+                        //"tasklist /FI \"PID eq " + pidString + "\" /FO csv /NH | find \"\"\"" + pidString + "\"\"\" > nul",
+                        //"if not errorlevel 1 (",
+                        //"    timeout /t 1 > nul",
+                        //"    goto :wait_for_close",
+                        //")",
+
+                        // Kill the process, why did we wait for it in the first place? - this is easier and less error prone
+                        "taskkill /PID " + pidString + " /F",

```

Comment:
I want to test this out thoroughly before pushing, but it sounds okay

Reply:
Update: the changes made here can just be reverted since #29 removes this segment of code completely.